jwst/datamodels/schemas/miri_ramp.schema.yaml

```diff
@@ -1,5 +1,6 @@
 allOf:
 - $ref: core.schema.yaml
+- $ref: wcsinfo.schema.yaml
 - $ref: bunit.schema.yaml
 - type: object
   properties:
```

Comment:
I'm not sure how these are supposed to work. I know that for models like MultiSlitModel, where each slit instance contains a SCI extension, it worked fine to have keywords assigned to the SCI header in the schema, in which case you get unique values of those keywords in each SCI extension in the FITS file. It looks like models like MultiExposure are built the same way, using a list of "data" attributes, each one of which maps to a SCI extension, and hence it should be possible to have the WCS keywords defined to be in the SCI extension and they should show up - with potentially unique values - in each SCI extension.